changelog_unreleased/javascript/16163.md

```diff
@@ -0,0 +1,32 @@
+#### Implement multilineObject config option (#16163 by @pauldraper)
+
+Prettier has historically done semi-manual formatting of multi-line JavaScript object literals.
+
+Namely, an object is kept on multiple lines if there is a newline prior to the first property, even if it could fit on a single line.
```

Comment:
```suggestion
Namely, an object is kept on multiple lines if there is a newline prior to the first property, even if it could fit on a single line. See [Multi-line objects](rationale.md#multi-line-objects) for more details.
```